examples/tomo/xray_trafo.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2014, 2015 The ODL development group
```

Comment:
I'd put it somewhere under tests, but preferably without storing images if at all possible. I forsee an explosion of these images if we start doing it. An alternative is to use a very small example and explicitly write the expected result in the test.